docs/UserGuide.md

```diff
@@ -3,44 +3,122 @@ layout: page
 title: User Guide
 ---
 
-SoConnect is a **desktop app for managing contacts and tasks**. It aims to help NUS SoC students stay better connected to their school life, in terms of social connections and school tasks. SoConnect is also **optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, SoConnect can **get your contact and task management done faster than traditional GUI apps.**
+## 1. Welcome to SoConnect's User Guide!
 
-* Table of Contents
-{:toc}
+Everything you need to know about **SoConnect**, an ultimate solution for your everyday contacts and tasks management problem, is finally here!
+
+If you are an **[SoC](#soc) student**, there is a high chance you are probably stressing over lack of social interactions and mounting assignments right now. Well, look no further! SoConnect is built to help you stay better connected to school life by doing the tedious management task, so you can focus on **creating real friends and finishing schoolwork quickly**. 
```

Comment:
It looks good, but I am actually unsure about the `a/an` part. Reason I put an is because while it starts with S, but since we pronounce each letter separately like as-ou-si (SOC), the `as` part sounds like it starts with vowel `a` and hence the an. It's the same reasoning with why we use `a university` instead of `an university` (because uni sounds loke yuni) or `an hour` instead of `a hour`.